db/models.py

```diff
@@ -17,7 +20,7 @@ def __str__(self):
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, db_index=True)
```

Comment:
It works, but in videos Dania said that it`s better to set index using Meta class.